React/wbooks/src/SignUp/style.scss

```diff
@@ -0,0 +1,55 @@
+@import "./../Components/app.scss";
+
+.login-container {
```

Comment:
some of these styles are exactly the same as in Login/styles.scss. This probably means that you could extract to a generic component (for example, Input) and move the style there, also allowing to override the style if necessary.

This way you avoid repeating styles all over the app

```
import styles from './styles.css';

function Input({ className, otherProp, ...props }) {
  <input className={`${className} ${styles.defaultFormInput}`} {otherProp} {...props} />
}

export default Input
```